presto-tests/src/test/java/com/facebook/presto/execution/resourceGroups/db/H2TestUtil.java

```diff
@@ -163,12 +163,12 @@ private static void setup(DistributedQueryRunner queryRunner, H2ResourceGroupsDa
         dao.insertResourceGroup(4, "adhoc-${USER}", "1MB", 3, 3, 3, null, null, null, null, null, null, null, 3L, TEST_ENVIRONMENT);
         dao.insertResourceGroup(5, "dashboard-${USER}", "1MB", 1, 1, 1, null, null, null, null, null, null, null, 3L, TEST_ENVIRONMENT);
         dao.insertResourceGroup(6, "no-queueing", "1MB", 0, 1, 1, null, null, null, null, null, null, null, null, TEST_ENVIRONMENT_2);
-        dao.insertSelector(2, "user.*", "test", null);
-        dao.insertSelector(4, "user.*", "(?i).*adhoc.*", null);
-        dao.insertSelector(5, "user.*", "(?i).*dashboard.*", null);
-        dao.insertSelector(4, "user.*", null, CLIENT_TAGS_CODEC.toJson(ImmutableList.of("tag1", "tag2")));
-        dao.insertSelector(2, "user.*", null, CLIENT_TAGS_CODEC.toJson(ImmutableList.of("tag1")));
-        dao.insertSelector(6, ".*", ".*", null);
+        dao.insertSelector(2, "user.*", "test", null, 10_000);
```

Comment:
Why the priority values in this case are 100, 1000,... etc? (instead of 1, 2, 3, 4 in previous examples?)

Reply:
In `testSelectorPriority()` unit test I insert a selector that has a priority value between two of these selectors, so I left some gaps here.